index.html

```diff
@@ -0,0 +1,117 @@
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li id="campaign"><a href="#">Campaign</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="welcome">
+            <div class="welcome-header">
+                <p>"Hello! Developers"</p>
+                <h1>Web Developers Conference 2023</h1>
+            </div>
+            <p>
+                Explore our growing collection of courses on key web design and development subjects.
+                An industry expert has written each course, helped by members of the Learn Web team.
+                Follow the modules sequentially, or dip into the topics you most want to learn about.
+            </p>
+            <h2 class="date">2023.03.20(Mon)~24(Fri)</h2>
+            <p class="location">@ Moscone Center in San Francisco, CA</p>
+        </section>
+        <section class="program">
+            <h2>Main Program</h2>
+            <div class="guide_bar"></div>
+            <div class="program-container">
+                <div class="program-card">
+                    <img src="./images/program_icon_01.png" alt="Lecture">
+                    <h2>Lecture</h2>
+                    <p class="program-info">Follow the modules sequentially, or dip into the topics you most want to learn.</p>
+                </div>            
+                <div class="program-card">
+                    <img src="./images/program_icon_02.png" alt="Lecture">
+                    <h2>Exhibition</h2>
+                    <p class="program-info">Appreciate various creations applying CC license of artists organized from Art Center Nabi.</p>
+                </div>
+                <div class="program-card">
+                    <img src="./images/program_icon_03.png" alt="Lecture">
+                    <h2>Forum</h2>
+                    <p class="program-info">Have the time to share your thoughts and opinions with experts for each topic.</pc>
+                </div>            
+                <div class="program-card">
+                    <img src="./images/program_icon_04.png" alt="Lecture">
+                    <h2>Workshop</h2>
+                    <p class="program-info">Try creating your own work using open source license rather than just watching.</p>
+                </div>            
+                <div class="program-card">
+                    <img src="./images/program_icon_05.png" alt="Lecture">
+                    <h2>Ignite</h2>
+                    <p class="program-info">Get Opportunities to network with CC affiliates around the world, also afer the summit.</p>
+                </div>
+            </div>
+            <a href="#" id="program-join">Register WDC 2023</a>
+            <a href="#" id="whole-program">SEE THE WHOLE PROGRAM</a>
+        </section>
+        <section class="speakers">
+            <h2>Featured Speakers</h2>
+            <div class="guide_bar"></div>
```

Comment:
- [ ] [REQUIRED] Good job so far! After `1380px` on the desktop version, the first speaker on the left side is `misaligned` with the rest of the speakers, please fix this it and itwould be great 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/217883142-42b41678-9403-4b2f-b2b1-0e6d4d0d1ff9.png)

Reply:
So Nice my friend 😃